### PR TITLE
chore: allow manual initialization of the SDK

### PR DESCRIPTION
**Pull request by marandaneto**
## :bulb: Motivation and Context
Closes https://github.com/PostHog/posthog-flutter/issues/22
Closes https://github.com/PostHog/posthog-flutter/pull/113
Required for https://github.com/PostHog/posthog-flutter/pull/116

Automatic mode is still default so not a breaking change.

## :green_heart: How did you test it?
Running auto init and automatic mode on android and ios

## :pencil: Checklist

- [X] I reviewed the submitted code.
- [ ] I added tests to verify the changes.
- [ ] I updated the docs if needed.
- [X] No breaking change or entry added to the changelog.

### CODE REVIEW

**Comment by marandaneto** [comment]
@thisames if you'd like to review as well :)

**Comment by thisames** [comment]
> @thisames if you'd like to review as well :)

for me, everything is ok

File: android/src/main/kotlin/com/posthog/posthog_flutter/PosthogFlutterPlugin.kt [hunk 1]
```diff
@@ -14,57 +18,84 @@ import io.flutter.plugin.common.MethodChannel.MethodCallHandler
...
 /** PosthogFlutterPlugin */
-class PosthogFlutterPlugin : FlutterPlugin, MethodCallHandler {
-    /// The MethodChannel that will the communication between Flutter and native Android
-    ///
-    /// This local reference serves to register the plugin with the Flutter Engine and unregister it
-    /// when the Flutter Engine is detached from the Activity
+class PosthogFlutterPlugin :
+    FlutterPlugin,
+    MethodCallHandler {
+    // / The MethodChannel that will the communication between Flutter and native Android
```
Comment (pauldambra) [inline android/src/main/kotlin/com/posthog/posthog_flutter/PosthogFlutterPlugin.kt:24]:
```suggestion
    // / The MethodChannel that will be the communication between Flutter and native Android
```

??

**Review by pauldambra — APPROVED** [review]
👍

**Pull request merged**